opengrok-tools/src/main/python/opengrok_tools/utils/commandsequence.py

```diff
@@ -128,14 +128,14 @@ def run(self):
                         if not self.driveon:
                             self.logger.debug("command '{}' for project {} "
                                               "requested break".
-                                              format(self.name, command))
+                                              format(command, self.name))
```

Comment:
Was this a transposition bug (and below)? Should a brief mention about a fix be included in the commit message?

Reply:
It was.